cypress/e2e/User.Notification.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Notifications regarding followed Users
+  As a user
+  I want to follow other users
+  To get notified about them posting in the network
+
+  When absent or offline I get notified via e-mail
+
+  Background:
+    Given the following "users" are in the database:
+      | name               | slug               | email                 | password | id   |  termsAndConditionsAgreedVersion |
+      | Bob der Baumeister | bob-der-baumeister | moderator@example.org | 1234     | u2   |  0.0.4                           |
+      | Jenny Rostock      | jenny-rostock      | user@example.org      | 1234     | u3   |  0.0.4                           |
+      | Peter Pan          | peter-pan          | peterpan@example.org  | 1234     | up   |  0.0.4                           |
+    And "Bob der Baumeister" follows "Jenny Rostock"
+
+  Scenario: Get notified about new public post from followed user
+    Given "Jenny Rostock" creates a public post in the network
+    Then "Bob der Baumeister" receives "1" email about a new public post from "Jenny Rostock"
+    Then "Peter Pan" receives "0" email about a new public post from "Jenny Rostock"
```

Comment:
You always reference email addresses for email checks

```suggestion
    Then "moderator@example.org" receives "1" email about a new public post from "Jenny Rostock"
    Then "peterpan@example.org" receives "0" email about a new public post from "Jenny Rostock"
```

I actually think that referencing the name is better for the scenario readability, since I do not have to look up the email in the table above to understand who is getting informed. Checkwise the email might be way easier.